sdk/ai/azure-ai-projects/samples/agents/sample_agents_azure_ai_search.py

```diff
@@ -108,6 +109,19 @@
     project_client.agents.delete_agent(agent.id)
     print("Deleted agent")
 
+    # [START populate_references_agent_with_azure_ai_search_tool]
     # Fetch and log all messages
     messages = project_client.agents.list_messages(thread_id=thread.id)
```

Comment:
You should be able to call with `order=ListSortOrder.ASCENDING`, so it would return the list in chronological order, avoiding the need to reverse later.

Reply:
Fixed.